lib/Url.js

```diff
@@ -1,11 +1,11 @@
 import TLD_REGEX from './tlds';
 
 const ALLOWED_PORTS = '([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])';
-const URL_PROTOCOL_REGEX = '((ht|f)tps?:\\/\\/)';
+const URL_PROTOCOL_REGEX = '((ht|f)tps?:[\\/|\\\\]{2})';
```

Comment:
Are you sure it is the right way to do it?

I am not an expert on regex. But I think you might have to use `(` not `[` with disjunction. I think something like
`(?:\\/{2}|\\\\{2})` should do the job.

Reply:
Your change will accept `https:/\google.com` and `https:||google.com` as valid url not only `https:\\google.com`